src/codec_aom.c

```diff
@@ -778,11 +783,39 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
             aom_codec_control(&codec->internal->encoder, AOME_SET_CQ_LEVEL, cqLevel);
         }
 #endif
+    }
```

Comment:
Lines 774-785 should only be done when initializing the encoder or when `cfg->rc_min_quantizer` or `cfg->rc_max_quantizer` changed. If it takes more than five lines to implement this, just add a TODO comment.

Reply:
Done